### ISSUE TITLE
Can the photo album table view display album folders?

### ISSUE THREAD

**Issue by interstateone** [Polarrco/polarr_ios_android_support#13]
I'm not sure if this is supported by the iOS SDK, since I'm not sure if I've seen it in another 3rd party app yet. The photo album table view displays a flat list of all photo albums, and for my case this means this list has hundreds of rows. In the Photos app these albums are organized in a hierarchy of folders, and so having this same hierarchy reflected in Polarr would make it easier to find things in the same place as Photos and also would save a lot of scrolling through a flat list.

**Comment by din** [comment]
Hello,

Have you tried switching albums using the top bar (which says 'Camera Roll' by default)? It shows all the available for your device folders. Just click on it once and select an album which you wish to browse.

**Comment by din** [comment]
Or I didn't understand you correctly and you mean nested album folders?

**Comment by interstateone** [comment]
Correct. So e.g. Folder A contains Albums 1, 2 and 3. Photos.app will display the folder and tapping it will display the 3 albums. 3rd party apps seem to display a flat list of all albums instead of the hierarchy. This might be a limitation of the SDK, but I'm not sure myself.

**Comment by interstateone** [comment]
Just checked another app, Snapseed, and it seems to present albums hierarchically. So hit or miss with 3rd party apps I guess.
